Examples/MAX32670/SecureROM_BL_Host/README.md

```diff
@@ -0,0 +1,113 @@
+## Description
+
+The Analog Devices Inc. (ADI) secure microcontroller has special bootloader, it is called as "Secure ROM Bootloader"
+this bootloaders provide firmware update function by secure way. The security related staff is handle by SDK and SDK tools.
```

Comment:
Period and capitalization typo, each line is a sentence.

Reply:
Thanks updated.